### DISCUSSION TITLE
docs/v1.12.x/ops/delete-worker-node/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2588] [Announcements]
# docs/v1.12.x/ops/delete-worker-node/

https://kubeovn.github.io/docs/v1.12.x/ops/delete-worker-node/

**Comment by zbb88888** [comment]
遇到一个偶现问题：Chassis 本身陷入一个错误状态，在vsctl show的结果中显示有自己的vxlan 隧道 endpoint 对应的port，正常情况下不应该有。

这个情况下，除了完全按照该文档的操作步骤，可能出现即使删掉了chassis，但是chassis依然按照同名重新注册上来的情况。此时需要在保证关闭kubelet，containerd的情况下，然后手动将ovn-controller 关联的进程都kill掉，然后确保所有的ovn ovs配置都已清理掉。最后再重新将node加回，重建Chassis应该可以恢复。

**Reply by zbb88888** [reply]
Chassis 是ovn-controller维护的，kube-ovn-controller应该只是更新了两个字段到 chassis的external_ids中，node和vendor。

**Reply by oilbeater** [reply]
我试了一下，应该是 containerd 在 stop 的时候不会停止已有的容器，我加个命令来处理一下这个情况
